scss/_tables.scss

```diff
@@ -187,3 +187,19 @@
     }
   }
 }
+
+
+// Fixed header
+//
+// Table header with the `.thead-fixed` class applied fix its cells on top
+// allowing only <tbody> to scroll.
+
+.table {
```

Comment:
This should be move into the previous block. No need to duplicate the selector. See the lint error.

Reply:
That's not a fix.

Reply:
Rather than move the block upwards I added the
```// stylelint-disable-next-line no-duplicate-selectors```
comment line right before it.

Reply:
He's saying not to do that Stylelint override, and instead move this into an existing `.table` ruleset.

Reply:
Done.